src/charm_state.py

```diff
@@ -1023,7 +791,7 @@ class CharmState:  # pylint: disable=too-many-instance-attributes
         charm_config: Configuration of the juju charm.
         is_metrics_logging_available: Whether the charm is able to issue metrics.
         proxy_config: Proxy-related configuration.
-        instance_type: The type of instances, e.g., local lxd, openstack.
+        instance_type: The type of instances, e.g., openstack.
```

Comment:
Do we still need this?

Reply:
It helped me with the removing, but you are right, it is time to detach fromt he InstanceType. Removed everywhere.